src/app/python/core.py

```diff
@@ -559,7 +570,39 @@ def _task_create_archive(self, path, caller, response):
 
     # Export file
     response.put(archive_path)
-    
+
+  @task_time(microseconds=1)
+  def _task_write_to_disk(self, caller):
```

Comment:
This whole task could be done in the C++ module ... @maym2104  had already prepared everything.
Since "developping" is complete for now, It can remain the same. However, it should be done in C++